tools/travis_script_osx

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash -ex
+#
+# this script is run on travis for the script stage of mac builds
+#
+
+QMAKE=${QMAKE:-qmake}
+LUPDATE=${LUPDATE:-lupdate}
+LRELEASE=${LRELEASE:-lrelease}
+MACDEPLOYQT=${MACDEPLOYQT:-macdeployqt}
+VERSIONID=${VERSIONID:-$(git rev-parse --short HEAD)}
+
+# build and test the CLI
+$QMAKE GPSBabel.pro && make && make check
+
+# build the GUI
+pushd gui
+$QMAKE app.pro && make
+$LUPDATE app.pro
```

Comment:
n.b. lupdate and lrelease used to be in the Mac build because that was what I built most. This ensured the translations were up to date when I released.

Reply:
They will be up to date when travis creates the dmg, but the updated files are not committed.